destalinator.py

```diff
@@ -66,17 +60,11 @@ def channel_minimum_age(self, channel_name, days):
         age = age / 86400
         return age > days
 
-    def debug(self, message):
-        self.logger.debug(message)
-        message = "DEBUG: " + message
-        if self.output_debug_to_slack_flag:
-            self.log(message)
-
     def flush_channel_cache(self, channel_name):
         """Flush all internal caches for this channel name."""
         cid = self.slacker.get_channelid(channel_name)
         if cid in self.cache:
-            self.debug("Purging cache for {}".format(channel_name))
+            self.logger.debug("Purging cache for {}".format(channel_name))
```

Comment:
Minor thing, but now that these are logger calls, linters will complain that they should use `%s` formatting rather than `"{}".format(…)`.